### ISSUE TITLE
Site navigation accessibility issues

### ISSUE THREAD

**Issue by struthj** [SmarterApp/AP_ItemSampler#8]
- [ ] Headings 
A logical heading structure greatly enhances the accessibility of a web page. Headings are used for page navigation and orientation. Each page should generally have one first-level heading (`<h1>`), which should typically contain the document title or main heading for that page. All headings should describe a section of content. When viewed in isolation, the heading structure should serve as a table of contents for the document. As such, heading levels should not be skipped; though it is acceptable to have lower level headings (such as an `<h2>`) precede the `<h1>` if the page structure requires it. 
The "Sample Items" heading at the top of the page should be marked up as an `<h2> i`nstead of an `<h3>`. This will present a more predictable and consistent heading structure. Severity 3 

- [ ]  Missing skip link 
A "Skip to main content" link should be presented to allow keyboard and screen reader users to quickly navigate directly to the main content of the page. Without a skip link, keyboard and screen reader users must generally navigate a long list of navigation links, sub-lists of links, corporate icons, site searches, and other elements before ever arriving at the main content. More information is available at http://webaim.org/techniques/skipnav/. Note that this link can be hidden visually until it receives keyboard focus, but should become visible upon receiving keyboard focus to be accessible to sighted keyboard users (see http://webaim.org/techniques/css/invisiblecontent/#skipnavlinks). 
A "skip navigation" link appears in Chrome, but in our testing, it did not appear in Edge or Firefox. Ensure it appears in all major browsers. Severity 3
